guides/common/modules/proc_enabling-remote-execution.adoc

```diff
@@ -0,0 +1,13 @@
+[id="enabling-remote-execution_{context}"]
+= Enabling Remote Execution
+
+Use this procedure to enable remote execution on your {SmartProxyServer}.
+To learn more about remote execution, see {ManagingHostsDocURL}Configuring_and_Setting_Up_Remote_Jobs_managing-hosts[Configuring and Setting Up Remote Jobs] in _{ManagingHostsDocTitle}_.
+
+.Procedure
```

Comment:
For `foreman-el` and `foreman-deb` think needs a prerequisite that REX is already enabled on the Server. Looking at the Installer Server guide I don't see a matching instruction you can link to. There it needs to be `{foreman-installer} --enable-foreman-plugin-remote-execution` or `{foreman-installer} --enable-foreman-plugin-remote-execution --enable-foreman-proxy-plugin-remote-execution-script` if you also want to run jobs from the Server itself.

For `katello` it's a dependency so already present. However, you may still need to enable the foreman-proxy-plugin on Server if you want to run jobs on it.